sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/NetAppAccountResource.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Threading;
+using Azure.Core;
+using Azure.ResourceManager.NetApp;
+using Azure.ResourceManager.NetApp.Models;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+namespace Azure.ResourceManager.NetApp
+{
+    public partial class NetAppAccountResource: ArmResource
+    {
+        [Obsolete("This method is obsolete and will be removed in a future release", false)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual AsyncPageable<NetAppVault> GetVaultsAsync(CancellationToken cancellationToken = default)
+        {
+            throw null;
+        }
+
+        [Obsolete("This method is obsolete and will be removed in a future release", false)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual Pageable<NetAppVault> GetVaults(CancellationToken cancellationToken = default)
+        {
+            throw null;
+        }
```

Comment:
We are removing this method in the new api-version, but does it still work in an old api-version?
The SDK supports api-version replacement, therefore it the API is still supported in an older api-version, we cannot just delete its implementation.
Is it still supported in an old api-version?

Reply:
yes, it is supported in older api-version.  How would we call using previous api-versoin ?

Reply:
Added back using older api-version